scalafix-tests/output/src/main/scala/test/explicitResultTypes/ImportAbstractOwner.scala

```diff
@@ -0,0 +1,22 @@
+package test.explicitResultTypes
+
+import test.explicitResultTypes.Package.{ Foo, Nested }
```

Comment:
I initially explored going for
```suggestion
import test.explicitResultTypes.Trait.{ Foo, Nested }

  val fooRef: Trait#Foo = foo
```
but the implementation looked more complicated, and less generic